addons/mail/static/src/components/attachment_list/attachment_list.xml

```diff
@@ -2,16 +2,16 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.AttachmentList" owl="1">
-        <div class="o_AttachmentList">
+        <div class="o_AttachmentList" t-attf-class="{{ classname }}">
```

Comment:
The more I progress, the more I wonder if we shouldn't make messaging component automatically patch this class into the DOM automatically.

Reply:
Would be great, but I don't know which satisfying hack could make this happen and just work.
Component templating looks too low-level and too rudimentary to make this possible.

Reply:
I mean just this.el.addClass (or equivalent in new syntax without el) in patched hook.

Reply:
I'll make test